app/views/schema.py

```diff
@@ -0,0 +1,22 @@
+from os import listdir
+from os.path import isfile, join
+
+from flask import Blueprint, jsonify
+from app.utilities.schema import load_schema_from_params
+
+schema_blueprint = Blueprint('schema', __name__)
+
+
+@schema_blueprint.route('/schemas/<eq_id>/<form_type>', methods=['GET'])
+def get_schema_json(eq_id, form_type):
+    schema = load_schema_from_params(eq_id, form_type)
+
+    return jsonify(schema.json)
+
+
+@schema_blueprint.route('/schemas', methods=['GET'])
+def list_schemas():
+    schema_path = "data/en"
+    onlyfiles = [f for f in listdir(schema_path) if isfile(join(schema_path, f)) and f.endswith(".json")]
```

Comment:
We already have something like that, its just a case of actually moving the schemas out. 
https://github.com/ONSdigital/eq-survey-register
Im wary of over-engineering given that its increasingly likely that we will move the schemas out into a private repo in the near future.

Reply:
Okay, how about returning the link HATEOAS stylie, like you do here then: https://github.com/ONSdigital/eq-survey-register/blob/master/api.go#L62. The survey-launcher shouldn't have the logic for parsing the filenames was my main point, but the response structure got mixed in with it.

Reply:
Yeh I did think about HATEOAS but then that leads to launcher using the url to launch the surveys, rather than `eq_id` and `form_type` which wont work as the schemas in runner don't define `eq_id` and `form_type` on them

Reply:
Okay. I guess my point rested on making this a longer term solution, but if it's not possible to know what that's going to look like, it's fine as it is :+1: